addons/hr_expense/models/hr_expense.py

```diff
@@ -1427,6 +1428,18 @@ def _check_can_approve(self):
     def approve_expense_sheets(self):
         self._check_can_approve()
 
+        for sheet in self:
+            if sheet.payment_mode == 'own_account' and not sheet.employee_id.sudo().bank_account_id:
+                action = {
+                    "view_mode": "form",
+                    "res_model": "hr.employee",
+                    "type": "ir.actions.act_window",
+                    "res_id": sheet.employee_id.id,
+                    'name': sheet.employee_id.display_name,
+                    'views': [(False, "form")],
+                }
+                raise RedirectWarning(_("Employee should have a bank account set."), action, _("Employee Form"))
```

Comment:
Confirmed with PO, as we're trying to get redirection buttons with more homogeneous names following the "Go to XXXXXX" pattern
```suggestion
                raise RedirectWarning(_("Employee should have a bank account set."), action, _("Go to Employee"))
```
cc @Laura-labe